docs/development/contributing.md

```diff
@@ -123,15 +123,15 @@ We are happy to get contributions that are no commits as well. There are many
 other things you could contribute to ILIAS:
 
 * **Ideas for new Features**: The development of ILIAS is driven by requirements
-  from the community. Contribute your ideas via [feature requests](http://www.ilias.de/docu/goto.php?target=wiki_5307&client_id=docu#ilPageTocA119).
+  from the community. Contribute your ideas via [feature requests](https://docu.ilias.de/goto.php?target=wiki_5307&client_id=docu#ilPageTocA119).
 * **Bug Reports**: We do our best, but ILIAS might contain bugs we do not know
-  yet. Check out how the ILIAS Community handles [bug reports](http://www.ilias.de/docu/goto.php?target=wiki_5307&client_id=docu#ilPageTocA115).
+  yet. Check out how the ILIAS Community handles [bug reports](https://docu.ilias.de/goto.php?target=wiki_5307&client_id=docu#ilPageTocA115).
 * **Information about Security Issues**: Check out how the ILIAS community
-  handles [security issues](http://www.ilias.de/docu/goto.php?target=wiki_5307&client_id=docu#ilPageTocA112). 
+  handles [security issues](https://docu.ilias.de/goto.php?target=wiki_5307&client_id=docu#ilPageTocA112).
   Reporter of security issues will also be named in the release notes.
 * **Time for Testing or Testcases**: We always need people that contribute
   testcases and perform them before new releases. Please have a look
-  [here](http://www.ilias.de/docu/goto_docu_pg_64423_4793.html) (German only).
+  [here](https://docu.ilias.de/goto_docu_pg_64423_4793.html) (German only).
   An English translation of the Tester Guide will be provided in summer 2017. If
   you have questions before that, do not hesitate to contact our test case
```

Comment:
Is this section still needed / valid? 2017 is long gone......

Reply:
I've removed the sentence with the year 2017 in it.